browser.js

```diff
@@ -34,20 +14,33 @@ function checkParameters (iterations, keylen) {
     throw new TypeError('Key length not a number')
   }
 
-  if (keylen < 0 || keylen > MAX_ALLOC) {
+  if (keylen < 0 || isNaN(keylen) || keylen > MAX_ALLOC) {
```

Comment:
in `lodash` `isNaN`: `value != +value`
https://github.com/lodash/lodash/blob/092f90d2fca959fa75d14cae8a2377a58bcb122d/lodash.js#L10534
or better will use `keylen !== keylen` ?